src/expressions/functions/builtInFunctions.identifiers.ts

```diff
@@ -31,9 +32,15 @@ const fnId: FunctionDefinitionType = function (
 	targetNodeSequence
 ) {
 	const targetNodeValue = targetNodeSequence.first();
+	if (!targetNodeValue) {
+		throw new Error('XPDY0002: context is absent, it needs to be present to use id function.');
```

Comment:
Can we also move this error the the `XPathErrors` with `XPDY0002: context is absent, it needs to be present to use ${feature}` as message. No need to update the other places where it's used but lets try to keep using this pattern for errors we add.

Reply:
ok